extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -341,6 +341,16 @@ private RunningDevService startContainer(DockerStatusBuildItem dockerStatusBuild
             LOG.debug("Not starting Dev Services for Keycloak as 'quarkus.oidc.provider' has been provided");
             return null;
         }
+        try {
+            for (String prop : ConfigProvider.getConfig().getPropertyNames()) {
```

Comment:
@gastaldi I'm not sure how I can use it, given the tenant ids are random strings like `1`, `2`.

Reply:
There is a map in `OIDCConfig.namedTenants`, maybe check if that is not empty?

Reply:
> Sure, might be an option, though it would likely require changing the whole container set up be runtime init

But this snippet is already being performed during runtime, isn't it? I don't think it would work if that iteration were performed during build time

Reply:
@gastaldi Yeah, as far as I recall, it works right now as long as a property like `quarkus.oidc.auth-server-url` is not retrieved from some ConfigSource like Vault, it probably will never be though :-). I'll check if accessing `OidcConfig` works, would be great for now if it will

Reply:
@gastaldi It does not work unless the whole build step is moved to runtime init, it fails to start devmode if `OidcConfig` is made available for ex as a method parameter, and it also requires a Recorder parameter.
It is a probably a right way forward anyway, but for now I'll just disable devservices for keycloak :-)